frontend/src/scenes/experiments/Experiment.tsx

```diff
@@ -1109,8 +1109,8 @@ export function Experiment(): JSX.Element {
                             >
                                 <div className="mt">
                                     <InsightContainer
-                                        disableHeader={experimentInsightType === InsightType.TRENDS}
-                                        disableTable={experimentInsightType === InsightType.FUNNELS}
+                                        disableHeader={true}
```

Comment:
Is there a particular reason we don't want to give the option of choosing a funnel display view?

<img width="372" alt="Screen Shot 2022-05-13 at 9 40 01 PM" src="https://user-images.githubusercontent.com/25164963/168406147-ce0fe963-2a04-4c84-9f4c-a8d865ce008c.png">

Reply:
Yep: It now also shows a date range which doesn't make a lot of sense anymore, and is confusing: you shouldn't be able to change the date ranges here. And in breakdown view, top to bottom is pretty hard to parse. And unnecessary, now that funnel view fits well